common/app/views/main.scala.html

```diff
@@ -81,6 +81,9 @@
                     @fragments.immersiveMainMedia(page)
                 </div>
             }
+            case page: model.ContentPage if (page.item.content.isRecipeArticle && false) => {
+                @fragments.recipeArticleHeader(page)
```

Comment:
I know you did this with me, while I was still confused about how this would be implemented, but since this fragment only has one line in it, just put that line here.